packages/relay/src/lib/services/ethService/ethCommonService/ICommonService.ts

```diff
@@ -31,4 +32,14 @@ export interface ICommonService {
   getLatestBlockNumber(requestIdPrefix?: string): Promise<string>;
 
   genericErrorHandler(error: any, logMessage?: string): JsonRpcError;
+
+  validateBlockHashAndAddTimestampToParams(params: any, blockHash: string, requestIdPrefix?: string): Promise<boolean>;
```

Comment:
Hmm, i'm not sure all of these need to be interface methods. If it's in here I'm guessing the goal is more than one Eth service will utilize it.
Notably in the long run eth.ts should be split across all the eth services.
We might see an abstract spin out and this interface reduce but this isn't a blocker right now as we juggle the two designs